### PR TITLE
Models & Unittest

### PR DESCRIPTION
**Pull request by dhita-irma**
Create Models & Unittest
- User (Inherit from Django Abstract User) 
- Post 
- UserFollowing
- Like

### CODE REVIEW

File: network/migrations/0005_like.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.1 on 2020-09-21 15:35
```
Comment (justin--) [inline network/migrations/0005_like.py:1]: is this suppossed to be committed?

File: network/views.py [hunk 2]
```diff
@@ -55,23 +62,43 @@ def post_detail(request, pk):
     #TODO: implement LIKE 
 
 
-@login_required
-def following(request):
-    # Query for requested data
+def follow_list(request, username, fol_type):
```
Comment (justin--) [inline network/views.py]: can you put a little bit of description for some of these, the names dont feel descriptive enough to know what they are for.

File: network/models.py [hunk 3]
```diff
@@ -5,6 +5,15 @@
 class User(AbstractUser):
     pass
 
+    def get_following_list(self):
```
Comment (justin--) [inline network/models.py]: why dont you use these in https://github.com/dhita-irma/cs50w-network/pull/2/files#diff-23386aa9223bd5e93909e1ec52be8856R77-R91

File: network/views.py [hunk 4]
```diff
@@ -55,23 +62,43 @@ def post_detail(request, pk):
...
     if request.method == 'GET':
-        for i in range(len(obj)):
-            following_list.append(obj[i].following_user.username)
-        data = {
-            "user": request.user.username,
-            "following": following_list
-        }
+
+        if fol_type == 'following':
+            fols = user.following.all()
```
Comment (justin--) [inline network/views.py]: i think you can just do return 
{
user: username
"following": [user.username for user.following_list()]
} 

and similar for the one below

File: network/views.py [hunk 5]
```diff
@@ -10,27 +11,33 @@
...
-
-
 def index(request):
-    return render(request, "network/index.html", {
-        'form': PostForm(), 
-    })
+    return render(request, "network/index.html")
+
 
+def post_list(request, feed):
```
Comment (justin--) [inline network/views.py:15]: i think its more important to unit test this logic
you can do it through using mocks or other tools
https://stackoverflow.com/questions/2036202/how-to-mock-users-and-requests-in-django

File: network/migrations/0005_like.py (hunk 1 above)
Comment (dhita-irma) [reply network/migrations/0005_like.py:1]: Yes

File: network/views.py (hunk 2 above)
Comment (dhita-irma) [reply network/views.py]: Okay, I added some comment

**Pull request merged**